lib/rules/avoid-leaking-state-in-ember-objects.js

```diff
@@ -47,7 +45,7 @@ module.exports = {
 
     return {
       CallExpression(node) {
-        if (!ember.isEmberComponent(node, filePath)) return;
```

Comment:
WIthout the filepath info, how do you differentiate between `Ember.Route.extend` and `$.extend`, `_.extend`, `myObj.extend`, etc?

Reply:
Agree, but we also don't want to miss anything, file path isn't a guarantee either.  Do you have a suggestion?

Reply:
I guess we could be smart and track the object we're calling `.extend` on. But I'm not sure if it's worth the effort.

Reply:
@krisseldon, I don’t think there are any guarantees when using static analysis to figure out what If an object is an instance of X. Just wanted to bring it up hoping to avoid common false positives.

Reply:
what if it just looked for $ or _ on the left side?